text/0000-embedded-refract.md

```diff
@@ -0,0 +1,361 @@
+- Start Date: 2015-08-29
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This document outlines a new serialization format for embedding Refract.
+
+# Motivation
+
+1. Our current formats either look like XML, the DOM, or Lisp
+1. People already "get" JSON
+1. Other serializations we have are verbose
+
+# Detailed design
+
+## Overview
+
+For this serialization, there are two basic concepts:
+
+1. There is a reserved property name of `refract`
+1. There is a Refract Object with properties:
+    - element
+    - meta
+    - attributes
+    - content
+
+All valid JSON that adheres to reserved property is valid Embedded Refract.
+
+## Basic JSON
+
+We'll start with a basic JSON example, which we'll say is a person and their bowling scores.
+
+```json
+{
+  "first_name": "John",
+  "last_name": "Doe",
+  "age": 28,
+  "scores": [150, 202, 145]
```

Comment:
I think we need to list out the mappings from JSON types to element names for the auto-conversion that happens here.